fairscale/utils/testing.py

```diff
@@ -303,7 +304,14 @@ def replacement(*args: Any, **kwargs: Any) -> None:
 
             error_queue = multiprocessing.get_context("spawn").SimpleQueue()
             if "OMPI_COMM_WORLD_RANK" in os.environ:
+                # TODO (Min): this global used to be assigned every time this file is imported.
+                #     I changed it to be assigned on first use. Should be the same, but I am not
```

Comment:
extra indent?

Reply:
yeah, I was doing 4-space indent, but I can unindent it. I don't really know which way is the standard.

Reply:
fixed